cms/wp/db/backup/cleanup.yml

```diff
@@ -0,0 +1,6 @@
+tasks:
+  wp:db:backup:cleanup:
```

Comment:
This task seems generic enough that it doesn't need to be WP-specific.

Reply:
This has been moved outside of the cms directory & scope.